injector/ProgramProcessor.py

```
@@ -52,14 +52,10 @@ def run(self):
                 source = f.read()
 
             currAst = ast.parse(source)
-            injector = LogInjector(currAst, ltMap, logTypeCount)
-
-            logTypeCount = injector.maxLtCount
+            injector = LogInjector(currAst)
```

Ruff (0.8.2)

Local variable `injector` is assigned to but never used

Remove assignment to unused variable `injector`

(F841)